packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/zoom.types.ts

```diff
@@ -2,7 +2,20 @@
 import { AxisId } from '../../../../models/axis';
 import type { ExtremumFilter } from './useChartCartesianAxis.types';
 
-export type ZoomData = { axisId: AxisId; start: number; end: number };
+export type ZoomData = {
+  /**
+   * The axis id of the zoomed axis.
```

Comment:
Nit: The first axis is redundant IMO. 

```suggestion
   * The id of the zoomed axis.
```